### DISCUSSION TITLE
ember instructions

### DISCUSSION THREAD

**Discussion by mcfiredrill** [tailwindlabs/tailwindcss#11323] [Help]
Trying the latest ember instructions
https://tailwindcss.com/docs/guides/emberjs

I get this error when opening the app
```
Uncaught Error: Cannot find module './app.css'
    at webpackMissingModule (app.js:11:50)
```

**Comment by NullVoxPopuli** [comment]
the instructions need updated,
here: 
![image](https://github.com/tailwindlabs/tailwindcss/assets/199018/b14f4ee9-887a-4b50-8945-ffd5b734d86a)


what you want to do is place your tailwind stuff in `app/styles.css`, or `app/assets/styles.css` and _not_ use the legacy css pippeline.

I think the tailwind instructions mimic these: https://discuss.emberjs.com/t/ember-modern-css/19614, which goes more in depth

**Comment by Woodsoniop** [comment]
Thanks for sharing. I found a lot of interesting information here. [MyBalanceNow](https://www.mybalancenow.page/)

**Comment by bertdeblock** [comment]
I followed the instructions again using Ember CLI v4.12.1 (latest at the time of writing), and they still work fine for me.

**Reply by NullVoxPopuli** [reply]
i wonder if op imported `app/app.css` which probably should work, and i got confused to, with the default `app/styles/app.css` <- tailwind should not go here
